cmd/cli/kubectl-kyverno/utils/filter/filter.go

```diff
@@ -16,10 +17,7 @@ type policy struct {
 }
 
 func (f policy) Apply(result api.TestResults) bool {
-	if result.Policy == "" {
```

Comment:
This logic has to stay, we only want to check for a match if the element is present.